src/qfs.ml

```diff
@@ -60,6 +60,7 @@ external exists : client -> string -> bool = "ml_qfs_exists"
 external is_file : client -> string -> bool = "ml_qfs_is_file"
 external is_directory : client -> string -> bool = "ml_qfs_is_directory"
 external create : client -> string -> excl:bool -> params:string -> file = "ml_qfs_create"
+external create2 : client -> string -> excl:bool -> Unix.open_flag -> bool -> params:string -> file = "ml_qfs_create2_bytecode" "ml_qfs_create2"
```

Comment:
meh on new function, better add optional parameters to existing `create` (and it will also provide a name for this new bool which I cannot guess what it means)